vividus-plugin-json/src/main/java/org/vividus/json/steps/JsonSchemaValidationSteps.java

```diff
@@ -72,6 +75,24 @@ public void validateJsonAgainstSchema(String json, String schema)
         assertValidationMessages(validationMessages);
     }
 
+
+    @Then("JSON `$json` is valid against openapi schema `$openApiSchema`")
+    public void validateJsonAgainstOpenApiSchema(String json, String schema)
+    {
+        JsonNode componentNode = jsonUtils.readTree(schema).get(COMPONENTS);
+        JsonNode schemaNode = componentNode.get("schemas").elements().next();
```

Comment:
something wrong here, I believe it's not required to parse schema

Reply:
this comment is not resolved: the key problem here is that only one schema is retrieved from spec, what if the object to validate was created following the second or any other subsequent schema?

Reply:
Tested, for correct work of step need to have main dto as first in `schemas`.
As decision - we can pass main dto name as parameter in step and handle in inside step or modify schema with help path json step.

Reply:
it won't work if message entity is an array of DTOs

Reply:
what is the main goal of introducing this step? how will `$json` be provided?

Reply:
Usually it has nested structure.
Will not work for case with separate dto for error - because in common errorDto has another  structure in comparison with main response/request body.

As I understand, main goal - to validate potential response/request body agains schema which is generated by OpenApi spec.